react/features/speaker-stats/middleware.ts

```diff
@@ -57,7 +72,22 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
 
         break;
     }
+
+    case ADD_TO_OFFSET: {
+        const state = getState();
+        const { timelineBoundary } = state['features/speaker-stats'];
+        const { right } = getTimelineBoundaries(state);
+        const currentDuration = getCurrentDuration(state) ?? 0;
+
+        if (Math.abs((right + action.value) - currentDuration) < 1000) {
```

Comment:
will be nice to move to const this 1000 ms (eg: DEFAULT_REFRESH_TIME or something more to business oriented)